lib/capistrano/tasks/slackify.cap

```diff
@@ -4,6 +4,7 @@ namespace :slack do
   task :notify_started do
     run_locally do
       set :time_started, Time.now.to_i
+      set :release_hosts, release_roles(:all).map(&:hostname).join("\n")
```

Comment:
it means this change? https://github.com/onthebeach/capistrano-slackify/commit/0b67a6c501f9b96103d5bae4ec5806db7026770c